client/src/shared/components/workflow/WorkflowDialog.tsx

```diff
@@ -176,7 +183,15 @@ const WorkflowDialog = ({
                                 <FormLabel>Label</FormLabel>
 
                                 <FormControl>
-                                    <Input {...field} />
+                                    <Input
+                                        {...field}
+                                        onKeyDown={(event) => {
+                                            if (event.key === 'Enter') {
+                                                handleSubmit(saveWorkflow)();
```

Comment:
Pogledaj moj komentar od maloprije, ali kao dodatak tome - ovakvo pozivanje funkcije - primjeti `()` na kraju je koncept koji se zove [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE) a to je zastarjela praksa koja se danas izbjegava.

U svakom slučaju pošto se koristi isti handler više od jedanput, pametno ga je izbaciti u varijablu/funkciju. Evo kako san ja to napravija:

```js
const handleOnKeyDown = (event: KeyboardEvent) => {
    if (event.key === 'Enter' && !event.shiftKey) {
        saveWorkflow();
    }
};
```

Super inicijativa, bravo!